app/controllers/admin/routes.py

```diff
@@ -194,3 +194,8 @@ def courseManagement(term = None):
                             approvedCourses = approved,
                             terms = terms,
                             term = term)
+
+
+@admin_bp.route('/matchVolunteerParticipant',methods = ['GET','POST'])
+def matchParticipants():
+    matchVolunteers()
```

Comment:
How are you passing nothing to this function?

Reply:
I did not need this here. We already have it in controllers for volunteers.py.